chemprop/models/mpn.py

```diff
@@ -169,9 +161,25 @@ def forward(self,
         :param features_batch: A list of numpy arrays containing additional features.
```

Comment:
I can't comment directly for some reason, but can you make sure to update the docstring for `batch` to indicate that the inputs will be lists of lists instead of lists?